pkg/kubelet/metrics/metrics.go

```diff
@@ -736,6 +737,20 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
+
+	// ContainerStartDuration tracks container start duration.
+	ContainerStartDuration = metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Subsystem:      KubeletSubsystem,
+			Name:           ContainerStartDurationKey,
+			Help:           "Duration in seconds to start a container.",
+			Buckets:        []float64{0.5, 1, 2, 3, 4, 5, 6, 8, 10, 20, 30, 45, 60, 120, 180, 240, 300, 360, 480, 600, 900, 1200, 1800, 2700, 3600},
```

Comment:
This is seems a bit too much.

My first question to find good bucket would be to know whether container start duration just includes telling the runtime to create the container or waiting for the runtime to start the container.

But even if kubelet is waiting for the runtime, 60 seconds to start a container already sounds like a lot, is it not?